mod/c_members_byDepartment/pages/members/members_by_dept.php

```diff
@@ -46,16 +47,17 @@
 $provinces['pov-man'] = 'Gouvernement du Manitoba';
 $provinces['pov-nb'] = 'Gouvernement du Nouveau-Brunswick';
 $provinces['pov-nfl'] = 'Gouvernement de Terre-Neuve-et-Labrador';
-$provinces['pov-ns'] = 'Gouvernement de la Nouvelle-�cosse';
+$provinces['pov-ns'] = 'Gouvernement de la Nouvelle-�cosse';
 $provinces['pov-nwt'] = 'Gouvernement du Territoires du Nord-Ouest';
 $provinces['pov-nun'] = 'Gouvernement du Nunavut';
 $provinces['pov-ont'] = "Gouvernement de l'Ontario";
-$provinces['pov-pei'] = "Gouvernement de l'�le-du-Prince-�douard";
-$provinces['pov-que'] = 'Gouvernement du Qu�bec';
+$provinces['pov-pei'] = "Gouvernement de l'�le-du-Prince-�douard";
```

Comment:
These � characters show up in lists.

Reply:
Not sure why this happened but fixed it where it did